cupyx/scipy/special/_logsumexp.py

```diff
@@ -0,0 +1,50 @@
+import cupy as cp
+
+
+__all__ = ["log_softmax"]
```

Comment:
nit: We conventionally use names that begin with an underscore for private things instead of `__all__` variable. Would you remove this line?

Reply:
Thanks for sharing. I've removed that line :)